website/data/api-docs-nav-data.json

```diff
@@ -440,7 +440,12 @@
       },
       {
         "title": "<code>/sys/config/ui/custom-messages</code>",
-        "path": "system/config-ui-custom-messages"
+        "path": "system/config-ui-custom-messages",
```

Comment:
I think this slipped in by mistake :)

Reply:
Yeah, don't know what I was thinking there. Fixing that now.